src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -39,8 +45,36 @@
      * Андрей - 600
      * София - 900</p>
      */
+    public static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Comment:
```suggestion
    public static final DateTimeFormatter FORMATTER = DateTimeFormatter.ofPattern("dd.MM.yyyy");
```

Reply:
constant's names should be in upper case

Reply:
done, thanks!